### DISCUSSION TITLE
How can i delete an app

### DISCUSSION THREAD

**Discussion by mrports** [Budibase/budibase#6662] [Help] [answered]
Hi
can anyone tell me how to delete an app from my account?
thank you

**Comment by shogunpurple** [comment, marked as answer]
Hi @mrports - first find the app you want to delete from the budibase portal and click "Manage". From there you can access the delete option from the top right of that menu.
<img width="343" alt="Screenshot 2022-07-12 at 15 59 02" src="https://user-images.githubusercontent.com/11256663/178521379-7af1bf60-af36-4fca-a9dc-b43501793011.png">

**Reply by mrports** [reply]
Thank you very much sir
